### DISCUSSION TITLE
Forcing aDetailer to run in FP32 mode

### DISCUSSION THREAD

**Discussion by brknsoul** [Bing-su/adetailer#172] [Q&A]
I've noticed that running in FP32 mode, aDetailer looks a lot better than in FP16 mode.

Is there a way to force aDetailer to run in FP32 mode (upcasting)?

**Comment by Bing-su** [comment]
1. the 2GB checkpoint is fp16, so it doesn't change at fp32.
2. adetailer's yolo and mediapipe models are not involved in image generation
3. There is already `--no-half-vae` or `--upcast-sampling` option
